apis/database.js

```diff
@@ -80,9 +112,35 @@ export async function addLocalLocation(locationData) {
 
   const createdAt = moment.utc(locationObject.created).toISOString();
   await executeTransaction(
-    'insert into locations (key, coordinateKey, createdAt, resources, status, uploaded) values (?, ?, ?, ?, ?, ?)',
-    [key, key, createdAt, resourcesString, status, 0]
+    'insert into locations (key, coordinateKey, createdAt, resources, status, uploaded, updated) values (?, ?, ?, ?, ?, ?, ?)',
+    [key, key, createdAt, resourcesString, status, 0, 0]
   );
 
   return locationObject;
 }
+
+export async function createOrUpdateDatabase() {
+  const version = await SecureStore.getItemAsync(DATABASE_VERSION_KEY);
+
+  await createDatabases();
```

Comment:
Having the call to update the value in create database means it will be overwritten. If anything bad happens it might think it is done migrating when it isn't. Is it worth worrying about?

Reply:
That's fair. I have moved the call to update the database version outside of the `createDatabases` function so that the new version key is only written after full completion of migration.